spec/lib/govuk_publishing_components/presenters/schema_org_spec.rb

```diff
@@ -53,7 +53,7 @@
 
       search_action = {
         "@type": "SearchAction",
-        "target": "http://www.dev.gov.uk/search/all?keywords={query}&organisations[]=ministry-of-magic",
+        "target": "http://www.dev.gov.uk/search/all?keywords={query}&organisations%5B%5D=ministry-of-magic",
```

Comment:
Would it be worth also adding a unit test for the `PotentialSearchActionSchema` class?

Reply:
I thought about it, but the schema bits are generally pretty trivial and they're exercised pretty well by the `schema_org_spec`.